test/tests/ver-1dc/ver-1dc_base.i

```diff
@@ -0,0 +1,165 @@
+# This is the base input file for the ver-1dc case.
+# This input file is meant to be included into the ver-1dc.i input file
+# and into the ver-1dc_mms.i input file to be full.
+# Its purpose is to centralize the capability common to the two cases
+# within one file to minimize redundancy and ease maintenance.
+# It is not meant to be run on its own.
+
+# cl = 3.1622e18 # atom/m^3
+# N = 3.1622e22 # atom/m^3
+epsilon_1 = ${units 100 K}
+epsilon_2 = ${units 500 K}
+epsilon_3 = ${units 800 K}
+temperature = ${units 1000 K}
+# nx_num = 1000 # (-)
+trapping_site_fraction_1 = 0.10 # (-)
+trapping_site_fraction_2 = 0.15 # (-)
+trapping_site_fraction_3 = 0.20 # (-)
+# trapping_rate_coefficient = 1e15 # 1/s
+# release_rate_coefficient = 1e13 # 1/s
+diffusivity = 1 # m^2/s
+# simulation_time = 60 # s
+# time_interval_max = 0.3 # s
```

Comment:
Lots of commented parameters here. Given some of these might be in the "complete" inputs is there a particular reason to leave them here, or should they be removed? Perhaps the latter.

Reply:
Thanks for pointing that out, it was me trying things out when consolidating the different input files. 
It should indeed be deleted.